.github/workflows/reusable-build-test-release.yml

```diff
@@ -1188,6 +1258,26 @@ jobs:
           path: |
             ${{ needs.setup.outputs.directory-path }}/diag*
 
+  knowledge-tests-report:
+    needs: run-knowledge-tests
+    runs-on: ubuntu-latest
+    if: ${{ !cancelled() && needs.run-knowledge-tests.result != 'skipped' }}
+    steps:
+      - name: Download all summaries
+        uses: actions/download-artifact@v4
+        with:
+          pattern: summary-ko*
+      - name: Combine summaries into a table
+        run: |
+          echo "| Job | Total Tests | Passed | Failed  | Errors| Skipped | Report Link" >> "$GITHUB_STEP_SUMMARY"
```

Comment:
I'd unify column names, i.e. `Total Tests`, `Passed Tests`... as it is done for unit tests table (or the other way - change names in unit tests table, to match  this format`Total Tests`,  `Passed`...).
But if you think it's ok as it is now I am good.

Reply:
For the sake of consistency, lets do it.